tests/nfv/prepare_env.pm

```diff
@@ -25,30 +25,39 @@ use strict;
 use utils;
 use lockapi;
 use mmapi;
-use serial_terminal 'select_virtio_console';
+
+my $host1 = get_required_var('NFVTEST_IP1');
+my $host2 = get_required_var('NFVTEST_IP2');
+
+sub get_trafficgen_ip {
+    my $my_ip = script_output(q(ip -4 addr show eth0 | egrep inet|awk '{print $2}'|cut -d/ -f1));
+    return $host2 if ($my_ip eq $host1);
+    return $host1;
+}
 
 sub run {
-    my $vsperf_repo = "https://gerrit.opnfv.org/gerrit/vswitchperf";
-    my $dpdk_repo   = "http://dpdk.org/git/dpdk";
-    my $host1       = get_required_var('NFVTEST_IP1');
-    my $host2       = get_required_var('NFVTEST_IP2');
-    my $children    = get_children();
-    my $child_id    = (keys %$children)[0];
+    my $vsperf_repo   = "https://gerrit.opnfv.org/gerrit/vswitchperf";
+    my $dpdk_repo     = "http://dpdk.org/git/dpdk";
+    my $trafficgen_ip = get_trafficgen_ip();
+    my $children      = get_children();
+    my $child_id      = (keys %$children)[0];
 
-    select_virtio_console();
+    select_console 'root-ssh';
```

Comment:
I wonder about this change. Why select "root-ssh" or actually, what console is it? I do not understand the meaning. Why not just "root-console"?

Reply:
Hm OK, understandable, not convincing though, not your fault :)

Reply:
I will do a run with root-console just to make sure it also works and change it to your suggestion if so.